### ISSUE TITLE
[MAJOR] Memory leak — OrbitalCameraSystem window event listeners never removed

### ISSUE THREAD

**Issue by ormidales** [ormidales/microgl#252]
**Catégorie** : bug
**Sévérité** : major
**Fichier(s) concerné(s)** : `src/core/ecs/systems/OrbitalCameraSystem.ts` (~l.85, ~l.100), `src/demos/camera.ts`, `src/demos/gltf.ts`

#### Description
`attach()` adds `mouseup` and `touchend` listeners on `window`. `detach()` exists but is never called in any demo's teardown handler. Every demo reload accumulates listeners indefinitely, leaking memory and producing ghost input events.

#### Exemple de code problématique
```ts
// camera.ts — no detach() call on scene teardown
scene.onDestroy = () => {
  world.destroy();
  // ❌ orbitalSystem.detach() missing
};
```

#### Solution suggérée
- Call `detach()` in every demo's teardown / `onDestroy` handler.
- Consider using `AbortController` to group all listeners and remove them in one call.
- Document the requirement in the system's JSDoc.

```ts
const controller = new AbortController();
window.addEventListener("mouseup", handler, { signal: controller.signal });
// teardown:
controller.abort(); // removes all listeners at once
```

#### Critères d'acceptation
- [ ] `detach()` called in all demo teardown paths
- [ ] No duplicate listeners after repeated scene reloads (verified via DevTools Event Listeners panel)
- [ ] JSDoc on `attach()` warns that `detach()` must be called on cleanup
